src/status_im/translations/da.cljs

```diff
@@ -98,28 +98,17 @@
    :browsing-open-in-web-browser          "Åben i web browser"
    :browsing-cancel                       "Annullér"
 
-   ;sign-up
-   :contacts-syncronized                  "Dine kontakter er synkroniseret"
-   :confirmation-code                     (str "Tak! Vi har sendt dig en sms med en bekræftelseskode. "
-                                               "Vær venlig at indtaste koden for at bekræfte dit telefonnummer")
-   :incorrect-code                        (str "Koden var desværre forkert. Prøv venligst igen")
+   ;;sign-up
    :phew-here-is-your-passphrase          "*Puha* det var hårdt, her er din kodesætning (passphrase), *skriv den ned og gem den sikkert!* Du skal bruge den for at kunne gendanne din konto."
    :here-is-your-passphrase               "Her er din kodesætning (passphrase), *skriv den ned og gem den sikkert!* Du behøver den for at gendanne din konto."
-   :phone-number-required                 "Klik her for at indtaste dit telefonnummer, så vil jeg finde dine venner"
    :shake-your-phone                      "Fundet en bug eller har du et foreslag. Blot ryst din telefon!"
    :intro-status                          "Chat med mig for at opsætte din konto og ændre dine indstillinger!!"
    :intro-message1                        "Velkommen til Status\n Tryk på denne meddelelse for at angive din adgangskode og komme i gang!"
    :account-generation-message            "Vent et øjeblik, jeg er nødt til at gøre nogle skøre beregninger for at generere din konto!"
    :move-to-internal-failure-message      "Vi skal flyyte nogle vigtige filer fra din eksterne til din interne hukommelse. For at gøre dette har vi brug for din tilladelse. Vi vil ikke bruge din eksterne hukkomelse i kommende versioner."
    :debug-enabled                         "Debug server has been launched! You can now execute *status-dev-cli scan* to find the server from your computer on the same network."
 
-   ;phone types
```

Comment:
Maybe we don't need to touch the translations at all? Don't see problem with having them there tbh, especially if we bring it back in future. Easier to just not render data and leave it be maybe?

Reply:
agree with @oskarth let's do not remove translations

Reply:
removed and moved to #3204 I told you there would be translations removed in phone command @flexsurfer :D